src/components/calcite-action/calcite-action.scss

```diff
@@ -1,48 +1,46 @@
 :host {
   @extend %component-host;
   display: flex;
-  background-color: var(--calcite-app-background-clear);
+  background-color: transparent;
 }
 
 :host([disabled]) {
   pointer-events: none;
 }
 
-@import "../../scss/includes/_component";
-
 .button {
-  font-family: var(--calcite-app-font-family);
+  font-family: var(--calcite-font-family);
```

Comment:
I would really prefer if we removed all references to this variable. `font-family` should be set to `inherit` in instances where you need to override a browser default. That way the font can cascade all the way through the dom tree with a single declaration on the `:root`.

Reply:
Me too...but dat dang ol' button likes to revert back to its own thing...at least in Chrome.

Reply:
Same thing happens with inputs.

Reply:
you have to set it to `inherit`: https://codepen.io/paulcp/pen/YzqGPRb

Reply:
🤦

Reply:
💥

Reply:
Thanks @paulcpederson. I made it less dumb.

Reply:
Nice! If you want to really go for it we could update that sweet little README section as well.

Reply:
Hmmm...thought I'd done that.